### PR TITLE
Skin Ozzie Dashboard With Tailwind

### PR DESCRIPTION
**Pull request by darkboywonder**
Merging this pull request adds tailwind styles to the dashboard. When merged, the dashboard should look like so:

ozzie diy_ 1

All of the styles exist within the tags `class` attributes for now. If you want to move them to the stylesheet, I have no problem doing those adjustments.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,12 @@
     },
     "devDependencies": {
         "axios": "^0.16.2",
-        "bootstrap-sass": "^3.3.7",
         "cross-env": "^5.0.1",
-        "jquery": "^3.1.1",
+        "glob-all": "*",
```
Comment (mattstauffer) [inline package.json]: Should we have specific version constraints on this and purgecss?

File: resources/views/dashboard.blade.php [hunk 2]
```diff
@@ -5,147 +5,147 @@
...
+        <title>Ozzie - Tighten</title>
     </head>
+
     <body>
-        <div class="container">
-            <div class="content">
-                <div class="title m-b-md">
-                    Ozzie - Tighten
-                </div>
+        <div class=" bg-white border-t-4 border-indigo relative z-10 shadow">
```
Comment (mattstauffer) [inline resources/views/dashboard.blade.php]: can drop this leading space

File: resources/views/svg/launch.blade.php [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<svg class="fill-current text-indigo-muted inline-block w-8 h-8"xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (mattstauffer) [inline resources/views/svg/launch.blade.php]: space before xmlns

File: package.json (hunk 1 above)
Comment (darkboywonder) [reply package.json]: yes, I will set these to the current version.

Comment (darkboywonder) [reply package.json]: Yes, I can add actual versions to these.

**Comment by darkboywonder** [comment]
Everything is updated. Merging.

**Comment by darkboywonder** [comment]
I will leave the branch just incase there is anything I missed. Otherwise, feel free to delete if you are satisfied with everything.

**Comment by mattstauffer** [comment]
@darkboywonder naw we can always fix stuff later

**Pull request merged**